AllCaps_fail.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<ElectionReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<Election>
+<CandidateCollection>
+<Candidate objectId='can-br-alckmin'>
+	<BallotName><Text language='en'>GERALDO ALCKMIN</Text></BallotName>
+</Candidate>
+</CandidateCollection>
+<ContestCollection>
+<Contest xsi:type="CandidateContest" objectId="cc-br1-pres-br">
+	<Name>PRESIDENTIAL ELECTION, ROUND 1, 2018</Name>
```

Comment:
Thanks for adding this test file, Raj. Can you please make sure that we have the schema required elements in this sample? Thank you!

Reply:
Schema required elements added to the sample file